app/services/generate_billing.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class GenerateBilling
+  def initialize(month, admin)
+    @month = month
+    return unless @month
```

Comment:
What is someone forgot to pass a month? You will skip generating billing silently until someone complains. 
Month looks like required parameter. There is no sense in action without it. raise exception if month is not specified.